### PR TITLE
Add all enums and descriptions known

### PR DESCRIPTION
**Pull request by luke-a-williams**
Justification for all of these fields can be found in our document repository:
https://dsdmoj.atlassian.net/wiki/spaces/HIA/pages/4421157581/Images

### CODE REVIEW

File: openapi.yml [hunk 1]
```diff
@@ -546,15 +546,56 @@ components:
...
+              * `SCAR` - Scar, 
+              * `MARK` - Mark, 
+              * `OIC` - Offence in Custody
         orientation:
           type: string
           example: FRONT
-          description: The orientation describes the secondary foci, scope, or outer focus of the image, which is to be the anatomical body part or angle of an image. In practise this is the exterior foci of what is captured in the 'view' field.
+          enum: [ANKLE, ARM, DAMAGE, EAR, ELBOW, FACE, FIGHT, FINGER, FOOT, FRONT, HAND, HEAD, INCIDENT, INJURY, KNEE, LEG, LIP, NECK, NOSE, SHOULDER, THIGH, TOE, TORSO]
+          description: >
+            Orientation is the scope focus; describing the scope or outer focus of the image, normally referring to the highest level object of interest within the bounds of the photo itself. This is normally a body part or an angle of photography, such as a photo of someone’s FRONT or ARM.
```
Comment (chubberlisk) [inline openapi.yml:564]:
```suggestion
            Orientation is the scope focus; describing the scope or outer focus of the image, normally referring to the highest level object of interest within the bounds of the photo itself. This is normally a body part or an angle of photography, such as a photo of someone’s facial view (`FRONT`) or arm (`ARM`).
```

File: openapi.yml [hunk 2]
```diff
@@ -546,15 +546,56 @@ components:
         view:
           type: string
           example: FACE
-          description: The view describes the "close-up" - primary foci, subject, or inner focus of the image. In practise this is the interior foci of what is captured in the 'orientation' field.
+          enum: [FACE,TAT,OTH,SCAR,MARK,OIC]
```
Comment (chubberlisk) [inline openapi.yml:549]:
```suggestion
          enum: [FACE, TAT, OTH, SCAR, MARK, OIC]
```

File: openapi.yml [hunk 3]
```diff
@@ -546,15 +546,56 @@ components:
...
+              * `KNEE` - Knee,
+              * `LEG` - Leg,
+              * `LIP` - Lip,
+              * `NECK` - Neck,
+              * `NOSE` - Nose,
+              * `SHOULDER` - Shoulder,
+              * `THIGH` - Thigh,
+              * `TOE` - Toe,
+              * `TORSO` - Torso
         type:
```
Comment (chubberlisk) [inline openapi.yml:589]: We're missing the `enum` property here! 👀

**Review by chubberlisk — APPROVED** [review]
Looks good to me, just missing the `enum` property for `type`. Once done feel free to merge!

File: openapi.yml (hunk 3 above)
Comment (luke-a-williams) [reply openapi.yml:589]: Oops!

**Pull request merged**